src/types/index.ts

```diff
@@ -22,6 +22,7 @@ export type LogLevelIOS = 'none' | 'verbose';
 export type LogLevelAndroid = 'none' | 'verbose' | 'error' | 'warning';
 
 export type DiscoverReadersParams = {
+  timeout: number;
```

Comment:
let's make this an optional param
```suggestion
  timeout?: number;
```